app/styles.module.css

```diff
@@ -10,7 +10,89 @@
 }
 
 .content {
-  max-width: 350px;
+  max-width: 360px;
   height: 100%;
   border: 1px solid black;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: space-around;
+}
+
+.postHeader {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 10px;
+}
+
+.usernameGroup {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+}
+
+.postFooter {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 12px;
+  margin-bottom: 10px;
+}
+
+.likesContainer {
+  display: flex;
+  align-items: center;
+  gap: 6px;
+}
+
+.separator {
+  border: none;
+  border-top: 1px solid #ddd;
+  margin: 10px 0;
+}
+
+.username {
+  margin-right: 110px;
+  font-weight: bold;
+}
+
+.likes {
+  margin-right: 200px;
+  font-weight: bold;
+}
+
+.date {
+  font-weight: 200;
+}
+
+.postImage {
+  width: 320px;
```

Comment:
This should be a `width: 100%`, so the image spans the full width of the container.